### ISSUE TITLE
Expose ivy's checkconsistency flag in sbt api

### ISSUE THREAD

**Issue by a-warner** [sbt/sbt#385]
Currently, the only way to set ivy's checkconsistency flag to false in sbt is to construct an ivy repository manually like:

``` scala
resolvers += {
  val r = new org.apache.ivy.plugins.resolver.IBiblioResolver
  r.setM2compatible(true)
  r.setName("example-repo")
  r.setRoot("http://example.org/repo/")
  r.setCheckconsistency(false)
  new RawRepository(r)
} 
```

It would be nicer to be able to set this flag on a per-repository basis with the sbt api.

See http://groups.google.com/group/simple-build-tool/browse_thread/thread/f8fef46504ad8cff/6bdf8b6dfb26a5ad#6bdf8b6dfb26a5ad for more details.

**Comment by shikhar** [comment]
+1

**Comment by harrah** [comment]
Fixed by #845.
